backend/api/utils.py

```
@@ -2,50 +2,35 @@
 from django.db.models import F
 
 def should_make_private(video):
```

Ruff (0.8.2)

Missing return type annotation for public function `should_make_private`

Add return type annotation: `bool`

(ANN201)